<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanvasExporter</class>
 <widget class="QDialog" name="CanvasExporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capture Canvas as Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>&amp;Height:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>heightLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="sizeInfoLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>900.0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="presetsComboBox">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="presetLabel">
     <property name="text">
      <string>&amp;Preset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>presetsComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sizeLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Image Size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="widthUnitComboBox"/>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="heightUnitComboBox"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="aspectCheckBox">
     <property name="text">
      <string>Enforce Camera's &amp;Aspect Ratio</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="resolutionLabel">
     <property name="text">
      <string>&amp;Resolution:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>resolutionLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="resolutionUnitComboBox"/>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="alphaCheckBox">
     <property name="text">
      <string>Export Alpha &amp;Channel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>&amp;Width:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>widthLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="heightLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="resolutionLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="widthLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>presetsComboBox</tabstop>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>widthUnitComboBox</tabstop>
  <tabstop>heightLineEdit</tabstop>
  <tabstop>heightUnitComboBox</tabstop>
  <tabstop>resolutionLineEdit</tabstop>
  <tabstop>resolutionUnitComboBox</tabstop>
  <tabstop>aspectCheckBox</tabstop>
  <tabstop>alphaCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanvasExporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanvasExporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
